src/applications/check-in/pre-check-in/tests/e2e/pages/Introduction.js

```diff
@@ -67,7 +67,7 @@ class Introduction {
     const theDateBefore = new Date(theDate.getTime());
     theDateBefore.setDate(theDate.getDate() - 1);
     cy.get('[data-testid="expiration-date"]').contains(
-      theDateBefore.toLocaleDateString(),
+      format(theDateBefore, 'M/dd/Y'),
```

Comment:
Would something like `format(subDays(appointmentTime, 1), 'M/dd/Y')` work here?

Reply:
ah yeah I like that. I also refactored the display to use this since it is a little cleaner than the way I was doing it there. Thanks @acrollet